src/test/java/teammates/test/pageobjects/InstructorCourseStudentDetailsEditPage.java

```diff
@@ -73,7 +77,11 @@ public InstructorCourseStudentDetailsEditPage submitUnsuccessfully(String studen
         if (comments != null) {
             fillTextBox(commentsTextbox, comments);
         }
-        submitButton.click();
+        if(teamName != null){
```

Comment:
@Sumei1009, leave a whitespace before the <code>(</code>, and after the <code>)</code>. Applies to every part of this pr. There might be other places in the code where this wasn't strictly adhered to, but we should keep to our coding standards.